build-aux/m4/ax_boost_thread.m4

```diff
@@ -67,13 +67,24 @@ AC_DEFUN([AX_BOOST_THREAD],
         [AC_LANG_PUSH([C++])
              CXXFLAGS_SAVE=$CXXFLAGS
 
-             if test "x$host_os" = "xsolaris" ; then
-                 CXXFLAGS="-pthreads $CXXFLAGS"
-             elif test "x$host_os" = "xmingw32" ; then
-                 CXXFLAGS="-mthreads $CXXFLAGS"
-             else
-                CXXFLAGS="-pthread $CXXFLAGS"
-             fi
+             case "x$host_os" in
```

Comment:
Have you sent these changes upstream? If so, can you link to the upstream commits/PR. I'd rather not modify the macros in our tree unless absolutely necessary.

Reply:
To be honest I didn't even know there was an upstream. I'll send a patch file (or whatever GNU people do nowadays) and report back here.

Reply:
Great, thanks. You should be able to open a PR against the [autoconf-archive](https://github.com/autoconf-archive/autoconf-archive) repo on GitHub.

Reply:
[Done](https://github.com/autoconf-archive/autoconf-archive/pull/211).

Reply:
[Merged upstream](https://github.com/autoconf-archive/autoconf-archive/commit/3eaf1697e4734e2ed2e9960fa755733e01042223).